presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/procedures/FlushCachesProcedure.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.jdbc.procedures;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
+import io.prestosql.plugin.jdbc.CachingJdbcClient;
+import io.prestosql.spi.procedure.Procedure;
+
+import javax.inject.Provider;
+
+import java.lang.invoke.MethodHandle;
+
+import static io.prestosql.spi.block.MethodHandleUtil.methodHandle;
+import static java.util.Objects.requireNonNull;
+
+public class FlushCachesProcedure
```

Comment:
This looks like overkill for `Add dedicated test for JDBC metadata caching`.

We should be able to test things like caching without affecting product's public parts, like procedures exposed to the user.